x-pack/plugins/security_solution_serverless/public/upselling/register_upsellings.tsx

```diff
@@ -19,17 +19,10 @@ import React, { lazy } from 'react';
 import { EndpointPolicyProtectionsLazy } from './sections/endpoint_management';
 import type { SecurityProductTypes } from '../../common/config';
 import { getProductAppFeatures } from '../../common/pli/pli_features';
-import investigationGuideUpselling from './pages/investigation_guide_upselling';
+import { investigationGuideUpselling } from './messages/investigation_guide_upselling';
+const ThreatIntelligencePaywallLazy = lazy(() => import('./pages/threat_intelligence_paywall'));
```

Comment:
Can we move the lazy loading logic outside this module? otherwise, it will start growing with different page-specific component logic. All pages/sections should export the already lazy-loaded components from their own file, and here just do a regular import and register. 
Also, they are all missing the `Suspense` wrapper.